logs/field_test.go

```diff
@@ -22,15 +21,19 @@ func TestField_String(t *testing.T) {
 		f     Field
 		want  string
 		panic bool
+		fail  bool
 	}
 
+	types := make(map[FieldType]bool)
+
 	tests := []test{}
 
-	for i := 0; i < int(EndType); i++ {
+	for i := 0; i <= int(EndType); i++ {
```

Comment:
EndType must be not included - it isn't real type, it is marker only and the EndType value will be never use in program.